pom.xml

```
@@ -176,11 +206,18 @@
                     <excludes>
                         <exclude>**/uz/enterprise/mytex/MytexApplication.class</exclude>
                         <exclude>**/uz/enterprise/mytex/audit/**/*</exclude>
-                        <exclude>**/uz/enterprise/mytex/config/**/*</exclude>
+                        <exclude>**/uz/enterprise/mytex/configs/**/*</exclude>
```

```suggestion
                        <exclude>**/uz/enterprise/mytex/config/**/*</exclude>
```
use singular nouns for packages